modules/transport.js

```diff
@@ -338,7 +338,7 @@ Transport.prototype.onSignature = function(signature, broadcast) {
 	if (broadcast && !__private.broadcaster.maxRelays(signature)) {
 		__private.broadcaster.enqueue(
 			{},
-			{ api: 'postSignatures', data: { signatures: [signature] } }
+			{ api: 'postSignatures', data: { signature } }
```

Comment:
Signature is not an array here, the form now changed from:
`signatures: ['example_signature']` to `signature: 'example_signature'`. 

Same will happen for transactions.

I think makes more sense to keep on passing signatures and transactions as arguments - arrays and change `object` property in broadcaster.self.routes from `transaction/signature` to `transactions/signatures`

Reply:
We are always calling enqueue with single instances of signature and transaction. Keeping the current changes fits better for now.